Basilisk-Zero/Assets/Scripts/Dialogue/YarnInteractable.cs

```diff
@@ -7,12 +7,22 @@
 public class YarnInteractable : MonoBehaviour {
     // internal properties exposed to editor
     [SerializeField] private string conversationStartNode;
+    // Speed of the NPC
```

Comment:
Might be a problem because we're going to have to delete this file. The switch to ink makes Yarn obsolete. I think if you make a new CS file with this code in it then it will be fine